src/components/AccountModule/useClientBlockNumber.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState, useEffect, useContext } from 'react'
```

Comment:
We may want to move this context up to `root/contexts/`. We most likely also want to combine this with `useSyncInfo` as `web3.eth.getBlock('latest')` already gives us the block number.